indy_node/server/pool_req_handler.py

```diff
@@ -34,6 +39,15 @@ def authErrorWhileAddingNode(self, request):
         if error:
             return error
 
+        dest = operation.get(TARGET_NYM)
```

Comment:
Actually it can be done as part of static validation, not dynamic one. See `doStaticValidation`.

Reply:
ok